lte/gateway/format_all.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
couple of notes that are more or less NITs:
* name: this script is only for c code, format_all does not reflect that
* is there a better folder for this script - dev-tools maybe?
* this script fails if clang format is not installed - case on my host, but it's installed in the mgma vm
  * should we have feed-back in this case?
* the script takes some time, but is very silent
  * should we have some start, finish and current folder logging (or even current file)?
* using `set -euo pipefail` is a good pattern

Reply:
Replying point by point:
- Unsusre, I think we could name it `format_c.sh` perhaps
- I was thinking about that, but I did not find a good option. `dev_tools.py` is file in the directory `format_all.sh` currently is in. The only other `dev_tools` folder is in `cwf`, which would be out of place.
- I added a check if clang is installed and the script now exits on a helpful message.
- I changed it so it prints out every file
- I added the pipefail line